### ISSUE TITLE
Menu Bar: add support for icons

### ISSUE THREAD

**Issue by mvysny** [vaadin/vaadin-flow-components#198]
Please add support for having icons in MenuItems. Probably blocked by https://github.com/vaadin/vaadin-menu-bar/issues/33

**Comment by web-padawan** [comment]
Closing in favor of vaadin/vaadin-menu-bar#33 where this is reported. 

Now when Flow components issues are transferred to the web components repos, we kindly ask you:

1. Please submit issues to the individual components repositories.
2. Please don't report same issue twice for web component and Flow component. One issue is enough.

We will create issue templates in the components repos to make it easier to report bugs and feature requests.
